react-app/src/index.css

```diff
@@ -33,6 +33,30 @@ code {
   float: right;
 }
 
+.topbar{
+  height: 153px!important;
+}
+
+.topbar-searchbar-wrapper{
+  position: relative!important;
```

Comment:
Use a more strict selector to overright the ant-design class attributes instead of `!important` (e.g.
```
.antd .topbar-searchbar-wrapper {
  position: relative
}
```

Reply:
That doesn't seem to work, but will look more into it

Reply:
yeah i remember trying things like what you're suggesting in the past and it not working @marsw42